src/index.ts

```diff
@@ -1,18 +1,19 @@
 import Koa = require('koa');
-import mongoose = require('mongoose');
 
-const app = new Koa();
+const MongoClient = require('mongodb').MongoClient;
+const uri = "mongodb+srv://test-user_1:<password>@cluster0-dojin.mongodb.net/test?retryWrites=true";
```

Comment:
> `<password>`

Summarizing conversation: we'll want to look to look at local password storage. Will do soon!

Reply:
More conversation: we'll want to the atlas db or a local db as per dev or production builds.